core/src/test/java/com/linecorp/armeria/client/EndpointTest.java

```diff
@@ -33,14 +33,16 @@ public void parse() {
         assertThat(foo.ipAddr()).isNull();
         assertThat(foo.ipFamily()).isNull();
         assertThat(foo.hasIpAddr()).isFalse();
+        assertThat(foo.toUri("none+http").toString()).isEqualTo("none+http://foo");
```

Comment:
Could you add a test case that @trustin mentioned which takes `http://www.badguys.com/` as a parameter?

Reply:
I add this test case at `toUri()` method.